src/CSET/recipes/surface_spatial_difference.yaml

```
@@ -1,5 +1,5 @@
 category: Surface Spatial Plot Difference
-title: $METHOD $VARNAME difference ($BASE_MODEL - $OTHER_MODEL)
+title: "$METHOD $VARNAME\n Difference ($BASE_MODEL - $OTHER_MODEL)"
```

```suggestion
title: "$METHOD $VARNAME\nDifference ($BASE_MODEL - $OTHER_MODEL)"
```
We don't want a space after the newline.